chainservice/builder.go

```diff
@@ -147,10 +171,6 @@ func (builder *Builder) createFactory(forTest bool) (factory.Factory, error) {
 		if forTest {
 			return factory.NewStateDB(factoryCfg, db.NewMemKVStore(), factory.RegistryStateDBOption(builder.cs.registry))
 		}
-		opts := []factory.StateDBOption{
-			factory.RegistryStateDBOption(builder.cs.registry),
-			factory.DefaultPatchOption(),
```

Comment:
`factory.RegistryStateDBOption(builder.cs.registry)` is common option, so keep it